### PR TITLE
[Update] What is Longview

### PR DESCRIPTION
**Pull request by rsyracuse**
Updates pre-existing docs for Cloud Manager Longview parity

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 70d54572ead5221a862f526f1875037df4f5ebad

https://deploy-preview-3086--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/longview/longview-app-for-apache/index.md [hunk 1]
```diff
@@ -71,46 +101,50 @@ To enable the Apache Longview app manually, follow these steps on your Linode vi
...
 You should now be able to see Longview data for Apache. If that's not the case, proceed to the [Troubleshooting](#troubleshooting) section at the end of this article.
 
 ## Viewing Statistics
 
 To see the output for the Longview Apache App:
 
-1.  Log in to the [Linode Manager](https://manager.linode.com/).
-2.  Select the **Longview** tab.
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+2.  Select the **Longview** sidebar.
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache/index.md]: Select the **Longview** link in the sidebar.

File: docs/platform/longview/longview-app-for-apache-classic/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+4.  Longview is designed to check the default location automatically. If you use the default location shown above, you should be done. Refresh the Longview Apache tab in the Linode Manager to verify that it's working now.
+5.  If you're not using the default location, you need to create a new file, `/etc/linode/longview.d/Apache.conf`, and set the `location` variable to match what you set in the Apache configuration file:
+
+    {{< file "/etc/linode/longview.d/Apache.conf" apache >}}
+location http://127.0.0.1/custom/location/path
+
+{{< /file >}}
+
+6.  Determine if an Apache virtual host configuration is interfering with requests to the mod_status location. Use a tool like `curl` or `wget` to request the server status location:
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache-classic/index.md]: mod_status should be in backticks

File: docs/platform/longview/longview-app-for-apache-classic/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+The **Disk IO** graph shows the amount of input to and output from the disk caused by Apache at the selected time. To see the total IO instead, visit the [Disks tab](/docs/platform/longview/longview/#disks).
+
+### Process Count
+
+The **Process Count** graph shows the total number of processes on your Linode spawned by Apache at the selected time. If you want to see more details, and how this stacks up against the total number of processes on your Linode, see the [Process Explorer tab](/docs/platform/longview/longview/#process-explorer).
+
+## Troubleshooting
+
+If you don't see Longview data for Apache, you'll instead get an error on the page and instructions on how to fix it. As a general tip, you can check the `/var/log/linode/longview.log` file for errors as well. You should also compare your mod\_status configuration file (typically `httpd.conf` or `status.conf`) to the example shown in Step 2 of the [Manual Configuration (All Distributions)](#manual-configuration-all-distributions) section of this article.
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache-classic/index.md]: mod_status should be in backticks

File: docs/platform/longview/longview-app-for-apache-classic/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+By default Longview uses port 80 for its automatic configuration. In the event you are experiencing problems you may need to edit the `/etc/apache2/ports.conf` file to use port 8080 or another non-standard port.
+
+### Autoconfigure Mod\_Status Popup
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache-classic/index.md]: I don't think we should capitalize Mod_Status

File: docs/platform/longview/longview-app-for-apache/index.md [hunk 5]
```diff
@@ -179,58 +213,58 @@ If you choose:
 -   **\<Yes\>**: the Longview tool will attempt to enable mod\_status, set the status page location, and restart Apache. This option is easier, but has the potential to disrupt your current Apache configuration. If you choose yes, and the configuration is successful, you should see output like the following:
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache/index.md]: mod_status in backticks

File: docs/platform/longview/longview-app-for-apache-classic/index.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+3.  Check the location for `mod_status`. The default location on Debian and Ubuntu systems is `http://127.0.0.1/server-status?auto` on localhost. In the Apache configuration file (typically `httpd.conf` or `status.conf`), this is designated with the lines:
+
+    {{< file "httpd.conf" apache >}}
+<Location /server-status>
+    SetHandler server-status
+
+{{< /file >}}
+
+    The `SetHandler server-status` line indicates that this is the location block for mod\_status. The location line itself sets the location.
```
Comment (andystevensname) [inline docs/platform/longview/longview-app-for-apache-classic/index.md]: mod_status in backticks

File: docs/platform/longview/what-is-longview/index.md [hunk 7]
```diff
@@ -277,182 +273,45 @@ The Overview tab shows all of your system’s most important statistics in one p
...
-
-2.  Then restart Longview with the `debug` flag:
-
-        sudo /etc/init.d/longview debug
+### Processes
+The Processes tab will list all running processes currently on your Linode, along with additional statistics including the UID of the user that has ownership of the process, the Max Count of duplicate processes, average I/O per process, average CPU percentage used per process, and average memory consumption per process. Additionally, at the top left of the page, you'll see a search bar you can use to filter the chart by the name of the process or the user that owns it.
 
-3.  When you're finished collecting information, repeat the first two steps to stop Longview and restart it again without the debug flag.
+You'll also see a number of graphs which display a history of resource consumption for the selected individual process. Historical values located on your graphs include CPU usage, RAM, the "count" of duplicate processes that are or were opened, and disk I/O. Additionally, the length of time you're observing with these graphs can be set in the dropdown menu found at the top right of the page and vary depending on your version of longview.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/platform/longview/what-is-longview/index.md]: fixed

File: docs/platform/longview/pricing/index.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-01-22
+modified_by:
+  name: Linode
+title: "Longview Pricing and Plans"
+contributor:
+  name: Linode
+---
+
+Longview is free for all customers for up to ten clients. You also have the option to purchase **Longview Pro** which includes additional analytics for an added cost. Longview's free version updates every 5 minutes and provides twelve hours of data history. Longview Pro gives you data resolution at 60 second intervals, and you can view a complete history of your Linode’s data instead of only the previous 30 minutes.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/platform/longview/pricing/index.md]: good call...updated to say 12 hours (you can also select a time resolution of 30 minutes, but the most is 12 hours)

File: docs/platform/longview/longview-app-for-apache/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache/index.md]: fixed

File: docs/platform/longview/longview-app-for-apache-classic/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache-classic/index.md]: this is classic manager guide...so disregarding

File: docs/platform/longview/longview-app-for-apache-classic/index.md (hunk 3 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache-classic/index.md]: this is classic manager guide...so disregarding

File: docs/platform/longview/longview-app-for-apache-classic/index.md (hunk 4 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache-classic/index.md]: this is classic manager guide...so disregarding

File: docs/platform/longview/longview-app-for-apache/index.md (hunk 5 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache/index.md]: fixed

File: docs/platform/longview/longview-app-for-apache-classic/index.md (hunk 6 above)
Comment (leslitagordita) [reply docs/platform/longview/longview-app-for-apache-classic/index.md]: this is classic manager guide...so disregarding

File: docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md [hunk 9]
```diff
@@ -0,0 +1,362 @@
...
+
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+2.  Select the **Longview** link in the sidebar.
+3.  Select the **Apache** tab.
+
+    ![The Longview Apache App.](longview_apache_stats.png)
+
+    You'll see the current version of Apache listed on the upper left hand side of the page.
+
+    Mouse over a data point to see the exact numbers for that time. With [Longview Pro](docs/platform/longview/pricing/), you can view older time periods for your data. The next sections cover the Longview Apache App in detail.
```
Comment (andystevensname) [inline docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md]: I think this needs a / at the beginning of the the link

File: docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md [hunk 10]
```diff
@@ -0,0 +1,362 @@
...
+
+- [Configuring Linode Longview for Apache](#debian-and-ubuntu-automatic-configuration).
+- [Interacting with the Apache data provided by Longview in the Linode Cloud Manager](#viewing-statistics).
+- [Troubleshooting Linode Longview for Apache](#troubleshooting).
+
+## Before you Begin
+
+In order to use the Longview App for Apache, ensure you have completed the following things:
+
+-  A Linode with [Apache installed and running]((/docs/web-servers/apache/)).
```
Comment (andystevensname) [inline docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md]: Remove extra set of parenthesis in the link

File: docs/platform/longview/what-is-the-linode-longview-app-for-nginx/index.md [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+1.  Click on the **Longview** link in the sidebar.
+1.  Select the Longview client instance whose NGINX data you'd like to view.
+1.  Select the **Nginx** tab.
+
+    ![The Longview Nginx App.](nginx-overview.png)
+
+    You'll see the current version of NGINX listed on the upper left-hand side of the page.
+
+    Mouse over a data point to see the exact numbers for that time. With [Longview Pro](docs/platform/longview/pricing/), you can view older time periods for your data. The next sections cover the Longview Nginx App in detail.
```
Comment (andystevensname) [inline docs/platform/longview/what-is-the-linode-longview-app-for-nginx/index.md]: I think this needs a beginning / to the link.

File: docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md (hunk 9 above)
Comment (leslitagordita) [reply docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md]: done

File: docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md (hunk 10 above)
Comment (leslitagordita) [reply docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md]: done

File: docs/platform/longview/what-is-the-linode-longview-app-for-nginx/index.md (hunk 11 above)
Comment (leslitagordita) [reply docs/platform/longview/what-is-the-linode-longview-app-for-nginx/index.md]: done

**Pull request merged**